src/openklant/components/klantinteracties/api/tests/test_notifications.py

```diff
@@ -0,0 +1,290 @@
+            "klantinteracties:internetaak-detail",
+            kwargs={"uuid": str(cls.internetaak.uuid)},
+        )
+        cls.data = {
+            "toegewezenAanActor": {"uuid": str(cls.actor.uuid)},
+            "aanleidinggevendKlantcontact": {"uuid": str(cls.klantcontact.uuid)},
+            "nummer": "1312312312",
+            "gevraagdeHandeling": "gevraagdeHandeling",
+            "toelichting": "toelichting",
+            "status": "verwerkt",
+        }
+
+    def test_send_notification_create_object(self, m):
+        with self.captureOnCommitCallbacks(execute=True):
+            response = self.client.post(self.list_url, self.data)
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        data = response.json()
+        m.assert_called_with(
+            {
+                "kanaal": "internetaken",
+                "hoofdObject": data["url"],
+                "resource": "internetaak",
+                "resourceUrl": data["url"],
+                "actie": "create",
+                "aanmaakdatum": "2024-02-02T00:00:00Z",
+                "kenmerken": {
+                    "nummer": data["nummer"],
+                    "gevraagdeHandeling": data["gevraagdeHandeling"],
+                    "toelichting": data["toelichting"],
+                    "status": data["status"],
+                },
+            }
+        )
+
+    def test_send_notification_update_object(self, m):
+        with self.captureOnCommitCallbacks(execute=True):
+            response = self.client.put(self.detail_url, self.data)
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        m.assert_called_with(
+            {
+                "kanaal": "internetaken",
+                "hoofdObject": data["url"],
+                "resource": "internetaak",
+                "resourceUrl": data["url"],
+                "actie": "update",
+                "aanmaakdatum": "2024-02-02T00:00:00Z",
+                "kenmerken": {
+                    "nummer": data["nummer"],
+                    "gevraagdeHandeling": data["gevraagdeHandeling"],
+                    "toelichting": data["toelichting"],
+                    "status": data["status"],
+                },
+            }
+        )
+
+    def test_send_notification_partial_update_object(self, m):
+        with self.captureOnCommitCallbacks(execute=True):
+            response = self.client.patch(self.detail_url, {"nummer": "123456789"})
+
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.json()
+        m.assert_called_with(
+            {
+                "kanaal": "internetaken",
+                "hoofdObject": data["url"],
+                "resource": "internetaak",
+                "resourceUrl": data["url"],
+                "actie": "partial_update",
+                "aanmaakdatum": "2024-02-02T00:00:00Z",
+                "kenmerken": {
+                    "nummer": data["nummer"],
```

Comment:
Let's explicitly test for the changed value here (applies to the other update tests too)
```suggestion
                    "nummer": "123456789",
```